app.R

```diff
@@ -0,0 +1,1123 @@
+                        inline = TRUE,
+                        choices = c('Gender',
+                                    'Diet'), 
+                        selected = 'Gender'
+                      )
+                    ),
+                    conditionalPanel(
+                      condition = "input.menu1 == 'Random'",
+                      sliderInput(
+                        inputId = 'slope1',
+                        label = 'Change the slope of Line A',
+                        min = -5,
+                        max = 5,
+                        value = 0,
+                        step = 1)
+                      ,
+                      sliderInput(
+                        inputId = 'slope2',
+                        label = 'Change the slope of Line B',
+                        min = -5,
+                        max = 5,
+                        value = 0,
+                        step = 1
+                      ),
+                      sliderInput(
+                        inputId = 'inter1',
+                        label = 'Change the intersection of Line A',
+                        min = -5,
+                        max = 5,
+                        value = 0,
+                        step = 1
+                      ),
+                      sliderInput(
+                        inputId = 'inter2',
+                        label = 'Change the intersection of Line B',
+                        min = -5,
+                        max = 5,
+                        value = 0,
+                        step = 1
+                      ),
+                      sliderInput(
+                        inputId = 'sample',
+                        label = 'Change the sample size',
+                        min = 100,
+                        max = 800,
+                        value = 100,
+                        step = 50
+                      )
+                    ),
+                    fluidRow(
+                      uiOutput('p'),
+                      align = "left"),
+                    conditionalPanel(
+                      condition = "input.menu1 != 'Random'",
+                      downloadButton(
+                        outputId = "downloadData", 
+                        label = "Download the Dataset",
+                        icon = shiny::icon("download")
+                      )
+                    )
+                  ),
+                  
+                  mainPanel(
+                    plotOutput('plot_gg'),
+                    tags$b(verbatimTextOutput('analysis1')),
+                    div(style = "text-align: center",
+                        bsButton(inputId = "game",
+                                 label = "Play!",
+                                 icon = icon("bolt"),
+                                 style = "danger",
+                                 size = "large")
+                    )
+                  )
+                )
+                ),
+        tabItem(tabName='game',
```

Comment:
When I tried opening the game page it did not work because it said no package called sortable was there. I think that the sortable package needs to be at the top with all of the packages because I had not downloaded that package before but once I did and put it at the top, the app worked.